src/libraries/Native/native-binplace.proj

```diff
@@ -11,16 +11,18 @@
   <!-- Ordering matters! Overriding GetBinPlaceItems and Build targets after the Sdk import. -->
   <Target Name="GetBinPlaceItems">
     <ItemGroup>
-      <BinPlaceItem Include="$(NativeBinDir)*.dll" />
-      <BinPlaceItem Include="$(NativeBinDir)*.pdb" />
-      <BinPlaceItem Include="$(NativeBinDir)*.lib" />
-      <BinPlaceItem Include="$(NativeBinDir)*.a" />
-      <BinPlaceItem Include="$(NativeBinDir)*.bc" />
-      <BinPlaceItem Include="$(NativeBinDir)*.so" />
-      <BinPlaceItem Include="$(NativeBinDir)*.dbg" />
-      <BinPlaceItem Include="$(NativeBinDir)*.dylib" />
-      <BinPlaceItem Include="$(NativeBinDir)*.dwarf" />
-      <FileWrites Include="@(BinPlaceItem)" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dll" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.pdb" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.lib" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.a" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.bc" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.so" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dbg" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dylib" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dwarf" />
+
+      <BinPlaceItem Condition="'$(binPlaceTestRuntimePack)' != 'true'" Include="@(_nativeItems)" />
```

Comment:
Nit: `b` should be upper case. `BinPlaceTestRuntimePack`

Reply:
Where is `_nativeItems` coming from?

Reply:
Good catch. It should be: 
```xml
<BinPlaceItem Condition="'$(binPlaceTestRuntimePack)' != 'true'" Include="@(NativeBinPlaceItems)" />
```